src/react-apps/applications/runtime/src/App.tsx

```diff
@@ -42,6 +43,9 @@ export default () => {
     FormResourceActions.fetchFormResource(
       `${window.location.origin}/runtime/api/textresources/${org}/${service}`,
     );
+    FormUserActions.fetchFormUser(
+      `${window.location.origin}/runtime/api/v1/profile/user`,
```

Comment:
Just as a little note, we should probably create utils/config for getting these kinds of urls in our solution, now we have to change them everywhere if the url potentialy changes.